### PR TITLE
[JENKINS-67967] "View build information" is not readonly

### PR DESCRIPTION
**Pull request by benebsiny**
See [JENKINS-67967](https://issues.jenkins-ci.org/browse/JENKINS-67967).

Image

When users who don't have permission enter the "View build information" pages, they are unable to edit the text.

### Proposed changelog entries

* Make "View build information" pages readonly for users who don't have permission.

### Proposed upgrade guidelines

N/A

### Submitter checklist

- [X] (If applicable) Jira issue is well described
- [X] Changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developer, depending on the change) and are in the imperative mood. [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
  * Fill-in the `Proposed changelog entries` section only if there are breaking changes or other changes which may require extra steps from users during the upgrade
- [X] Appropriate autotests or explanation to why this change has no tests
- [ ] New public classes, fields, and methods are annotated with `@Restricted` or have `@since TODO` Javadoc, as appropriate. 
- [ ] For dependency updates: links to external changelogs and, if possible, full diffs

### Desired reviewers

@mention

### Maintainer checklist

Before the changes are marked as `ready-for-merge`: 

- [ ] There are at least 2 approvals for the pull request and no outstanding requests for change
- [ ] Conversations in the pull request are over OR it is explicit that a reviewer does not block the change
- [ ] Changelog entries in the PR title and/or `Proposed changelog entries` are accurate, human-readable, and in the imperative mood
- [ ] Proper changelog labels are set so that the changelog can be generated automatically
- [ ] If the change needs additional upgrade steps from users, `upgrade-guide-needed` label is set and there is a `Proposed upgrade guidelines` section in the PR title. ([example](https://github.com/jenkinsci/jenkins/pull/4387))
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins-ci.org/issues/?filter=12146)).

### CODE REVIEW

**Review by timja — APPROVED** [review]
Looks right, un-tested

**Review by NotMyFault — COMMENTED** [review]
The preview buttons of the "Description" box are displayed still, which looks out of place, considering there's no message box shown to the user anymore.

**Comment by daniel-beck** [comment]
Does this need to be set twice, isn't once inside `<f:form>` enough?

**Comment by benebsiny** [comment]
@NotMyFault @daniel-beck I've fixed it. Thanks for your suggestions!

**Review by NotMyFault — APPROVED** [review]
Thanks for addressing the comments, looks good so far 👍🏻

File: core/src/main/resources/hudson/model/Run/configure.jelly
Comment (daniel-beck) [inline core/src/main/resources/hudson/model/Run/configure.jelly:37]: This change should be reverted.

While it looks off, it's consistent with job configuration forms and the global configuration, and should be addressed everywhere, ideally by changing `f:textarea`, instead of being a one-off change for this form field.

> Screenshot 2022-03-23 at 16 27 35
> Screenshot 2022-03-23 at 16 27 54

**Review by daniel-beck — CHANGES_REQUESTED** [review]

File: core/src/main/resources/hudson/model/Run/configure.jelly
Comment (benebsiny) [reply core/src/main/resources/hudson/model/Run/configure.jelly]: I reverted it. Maybe we can address this by opening another issue

**Review by timja — APPROVED** [review]

File: core/src/main/resources/hudson/model/Run/configure.jelly
Comment (NotMyFault) [reply core/src/main/resources/hudson/model/Run/configure.jelly]: We likely can address it later on and iterate across all previewEndpoint occurrences of `f:textarea`s.  I'm fine with dismissing my previous comment.

**Review by daniel-beck — APPROVED** [review]
Thanks!

**Comment by timja** [comment]
This PR is now ready for merge, after ~24 hours, we will merge it if there's no negative feedback.

Thanks!

**Pull request merged**
